plugin/kubernetes/setup_test.go

```diff
@@ -32,6 +33,7 @@ func TestKubernetesParse(t *testing.T) {
 			defaultResyncPeriod,
 			"",
 			podModeDisabled,
+			"",
```

Comment:
Please create a new parse test for just this option; this adds a whole bunch of empty strings for no purpose at all in this test.